src/fp.rs

```diff
@@ -321,12 +322,14 @@ impl Fp {
     }
 
     #[inline]
+    /// Performs the square root of an element in constant time.
```

Comment:
I think this is not constant time since it uses `pow_vartime`

Reply:
No. If you read `pow_vartime` it says "It's only variable in respect to the exponent" (which in this case is fixed (we just change the base when we call the fn).

Hence why we return a `CtOption` in the fn signature of `sqrt`.

Reply:
You're right! Good point